### DISCUSSION TITLE
Alpine Expression Error: Object.entries(...).flatMap is not a function

### DISCUSSION THREAD

**Discussion by thanhtoan1196** [alpinejs/alpine#2100] [7. Bugs (archived)]
I got this error,  how to fix it?

![image](https://user-images.githubusercontent.com/16433547/134191759-b18ee5d8-f5f0-4392-91d6-015dccad9d65.png)

**Comment by SimoTod** [comment]
Your browser doesn't support flatMap. Which one is it?

**Reply by thanhtoan1196** [reply]
@SimoTod here is the list

![image](https://user-images.githubusercontent.com/16433547/134193911-9f384ae5-bfcd-4cd8-87a8-8c7151f92006.png)

**Reply by thanhtoan1196** [reply]
how to workaround?

**Reply by SimoTod** [reply]
As a quick workaround, you can add `<script src="https://polyfill.io/v3/polyfill.min.js?features=Array.prototype.flatMap" defer></script>` before Alpine.

**Reply by thanhtoan1196** [reply]
ok, thank you
